lib/pbench/server/api/resources/__init__.py

```diff
@@ -528,6 +527,9 @@ def validate(self, json_data: JSON) -> JSON:
     def __contains__(self, key):
         return key in self.parameters
 
+    def __getitem__(self, key):
+        return self.parameters[key]
+
```

Comment:
I've thought of doing that several times since I had it wired for `in`, but when I added type checking for the `user` parameter I needed to *find* the `Parameter` that was `in` the schema.